i18n/sl.js

```diff
@@ -0,0 +1,95 @@
+export default {
+  lang: 'sl',
+  label: {
+    clear: 'Počisti',
+    ok: 'Vredu',
+    cancel: 'Prekliči',
+    close: 'Zapri',
+    set: 'Postavi',
+    select: 'Izberi',
+    reset: 'Obnovi',
+    remove: 'Odstrani',
+    update: 'Posodobi',
+    create: 'Ustvari',
+    search: 'Išči',
+    filter: 'Filtriraj',
+    refresh: 'Ponovno naloži'
+  },
+  date: {
+    days: 'Nedelja_Ponedeljek_Torek_Sreda_Četrtek_Petek_Sobota'.split('_'),
+    daysShort: 'Ned_Pon_Tor_Sre_Čet_Pet_Sob'.split('_'),
+    months: 'Januar_Februar_Marec_April_Maj_Junij_Julij_August_September_October_November_December'.split('_'),
+    monthsShort: 'Jan_Feb_Mar_Apr_Maj_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
+    firstDayOfWeek: 1, // 0-6, 0 - Nedelja, 1 Ponedeljek, ...
+    format24h: true
+  },
+  pullToRefresh: {
+    pull: 'Za ponovno nalaganje povleci navzdol',
+    release: 'Za ponovno nalaganje pusti',
+    refresh: 'Ponovno nalogam...'
+  },
+  table: {
+    noData: 'Ni dosegljivih podatkov',
+    noResults: 'Ne najdem ustreznic',
+    loading: 'Nalagam...',
+    selectedRows: function (rows) {
+      return rows === 1
+        ? '1 selected row.'
+        : (rows === 0 ? 'No' : rows) + ' selected rows.'
+    },
+    rowsPerPage: 'Vrstic na stran:',
+    allRows: 'Vse',
+    pagination: function (start, end, total) {
+      return start + '-' + end + ' od ' + total
+    },
+    columns: 'Stolpci'
+  },
+  editor: {
+    url: 'URL',
+    bold: 'Krepko',
+    italic: 'Ležeče',
+    strikethrough: 'Prečrtano',
+    underline: 'Podčrtaj',
+    unorderedList: 'Neoštevilčen seznam',
+    orderedList: 'Oštevilčen seznam',
+    subscript: 'Podpisano',
+    superscript: 'Nadpisano',
+    hyperlink: 'Hiper povezava',
+    toggleFullscreen: 'Toggle Fullscreen',
```

Comment:
Not translated.
"Toggle Fullscreen" => "Preklopi celoten zaslon"
Sorry, I don't have better/shorter translation for this.